packages/flutter/lib/src/animation/tween.dart

```diff
@@ -5,6 +5,7 @@
 import 'dart:ui' show Color, Size, Rect;
 
 import 'package:flutter/foundation.dart';
+import 'package:flutter/src/painting/gradient.dart';
```

Comment:
The normal way we do things is to import the sub-package, not the individual file:
```dart
import 'package:flutter/painting.dart';
```